assignments/prototype-refactor.js

```diff
@@ -1,9 +1,171 @@
 /* 
-
 Prototype Refactor
-
 1. Copy and paste your code or the solution from yesterday
+2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.     */
+
+class GameObject {
+    constructor(attributes) { 
+        this.createdAt = attributes.createdAt;
+        this.dimensions = attributes.dimensions;
+    }
+    destroy() { 
+        let destroyed = `Object was removed from the game`;
```

Comment:
```suggestion
        let destroyed = `${this.name} was removed from the game`;
```
This should be a dynamic name, not Object. And in order to make that happen the this.name property needs to be set in this class.